cnf/local/index.xml

```diff
@@ -1,3 +1,2 @@
-<?xml version="1.0" encoding="utf-8"?>
-<repository increment="0" name="Local" xmlns="http://www.osgi.org/xmlns/repository/v1.0.0">
-</repository>
\ No newline at end of file
+<?xml version='1.0' encoding='UTF-8'?>
+<repository xmlns="http://www.osgi.org/xmlns/repository/v1.0.0" name="Local" increment="1609514246505"/>
```

Comment:
Why are the local and release index files changing?

Reply:
bndtools automatically rewrites them when you open it once. I've removed the changes from the commit.